jsx/FilterableDataTable.js

```diff
@@ -24,40 +24,75 @@ class FilterableDataTable extends Component {
   constructor(props) {
```

Comment:
MAJOR collisions in this file mainly between the commits referenced below
main: 2372c4294cdfcdcce2843bbf6eb7a2d2df20a640, b9c25e8df2f029cdfb67c9698d4dbc290a060723
23: b52a8a9592041af2040fd9d121642619f53d9146

The reconciliation should be reviewed carefully as the bugfix and new feature both made major modifications